### PR TITLE
Replace getenv() with env()

### PR DESCRIPTION
**Pull request by alexweissman**
See https://github.com/userfrosting/UserFrosting/issues/1098. Not sure if we want to merge this in for a revision or save for another minor version.

### CODE REVIEW

File: app/sprinkles/core/src/Bakery/DebugCommand.php [hunk 1]
```diff
@@ -49,10 +49,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // Perform tasks & display info
         $this->io->definitionList(
-            ['UserFrosing version'  => \UserFrosting\VERSION],
```
Comment (lcharette) [inline app/sprinkles/core/src/Bakery/DebugCommand.php:52]: https://i.kym-cdn.com/entries/icons/mobile/000/030/710/dd0.jpg

**Comment by lcharette** [comment]
So `env` is a magic helper method of Laravel? Not sure I like that kind of magic, after the [`str_contain`](https://github.com/laravel/framework/issues/33045) fiasco. I wonder if they have a namespaces equivalent, like `Str::` method ?

If it doesn't change `.env` value, I don't see why we couldn't merge in hotfix ?

**Review by lcharette — APPROVED** [review]

**Review by lcharette — CHANGES_REQUESTED** [review]
Actually, there is a namespaced equivalent : https://github.com/illuminate/support/blob/master/Env.php 

I think it would be better long term...

**Comment by alexweissman** [comment]
I would say we should just follow the same pattern as Laravel, which uses the `env` helper in config files and tests: https://github.com/laravel/laravel/blob/8.x/config/database.php. I think this makes sense because the namespace could change in subsequent versions of Laravel but the helper method would still be independent of the namespace.

They do use `Env::` in namespaced code, if we wanted to be consistent there. Note that unlike the `str_` helpers, `env` has not been deprecated.

**Review by lcharette — APPROVED** [review]
Fair enough

**Comment by lcharette** [comment]
Note there was actually an issue with this PR that appeared after merge. See temp fix in https://github.com/userfrosting/UserFrosting/commit/926b9be79971a0d3c6d7b8765d4dc7dcb15f55dd

Error : https://github.com/userfrosting/UserFrosting/actions/runs/628080953

Looks like env doesn't use the loaded env right above : 
https://github.com/userfrosting/UserFrosting/blob/926b9be79971a0d3c6d7b8765d4dc7dcb15f55dd/app/sprinkles/core/src/ServicesProvider/ServicesProvider.php#L235-L245

**Pull request merged**